### PR TITLE
SPUI-1549 > Disclaimer pixel shoot

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        children: [
+          Container(
+            height: 27.0,
+            decoration: BoxDecoration(
+              gradient: LinearGradient(
+                stops: const [0.1, 1],
+                begin: Alignment.bottomCenter,
+                end: Alignment.topCenter,
+                colors: [
+                  Colors.white,
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart:30]: use simple colors

File: plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                  Colors.white.withOpacity(0.1),
```
Comment (VovaMedvediev) [inline plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart:31]: same here

File: plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart (hunk 1 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart:30]: done

File: plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/buttons/simple_floating_button_frame2.dart]: done

**Pull request merged**